lib/samples/filter_by_definition_expression_or_display_filter/filter_by_definition_expression_or_display_filter_sample.dart

```diff
@@ -54,21 +58,26 @@ class _FilterByDefinitionExpressionOrDisplayFilterSampleState
         child: Column(
```

Comment:
Does the SafeArea need a "top: false"? (So that the map goes all the way up under the notch when running with sample_runner.dart)

Reply:
I believe so!